Annex60/Fluid/Chillers/Carnot_y.mo

```diff
@@ -90,6 +85,24 @@ For a similar model that can be used as a heat pump, see
 revisions="<html>
 <ul>
 <li>
+January 2, 2017, by Filip Jorissen:<br/>
+Removed parameters
+<code>effInpEva</code> and <code>effInpCon</code>
+and updated documentation.
+This is for
+<a href=\"https://github.com/iea-annex60/modelica-annex60/issues/497\">
+issue 497</a>.
+</li>
+<li>
+August 8, 2016, by Michael Wetter:<br/>
```

Comment:
I am a bit confused whether this date is intentional chosen or if you forgot to edit it?